hcp-ui-templates/ec2-existing-vpc/main.tf

```diff
@@ -94,7 +115,34 @@ output "nomad_url" {
 output "hashicups_url" {
   value = "http://${module.aws_ec2_consul_client.public_ip}"
 }
-
+output "consul_export" {
+  value     = <<EOF
+  export CONSUL_HTTP_ADDR="${hcp_consul_cluster.main.consul_public_endpoint_url}"
+  export CONSUL_HTTP_TOKEN="${hcp_consul_cluster_root_token.token.secret_id}"
+  EOF
+  sensitive = true
+}
+output "nomad_export" {
+  value = <<EOF
+  export NOMAD_HTTP_AUTH="nomad:${hcp_consul_cluster_root_token.token.secret_id}"
+  export NOMAD_ADDR="http://${module.aws_ec2_consul_client.public_ip}:8081"  
+  sensitive = true
+  EOF
+}
 output "next_steps" {
-  value = "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token."
+  value = local.install_demo_app ? "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token." : null
+}
+output "ssh_to_client" {
+  value = local.ssh ? "ssh -i ${local_file.ssh_key.filename} ubuntu@${module.aws_ec2_consul_client.public_ip}" : null
+}
+output "connect_with_ssm" {
+  value = "aws ssm start-session --target ${module.aws_ec2_consul_client.host_id} --region ${local.vpc_region}"
+}
```

Comment:
I can see an argument either way, but why not keep these in the `how_to_connect` output below? (as new lines)

With one output snippet per connection method, we'll have a bunch of output strings. It seems like we could include each of these snippets in the how to connect section and reduce the clutter of the number of exported outputs

Reply:
I've just tested this out and it is indeed nicer, however it is a big output snippet which i guess is fine. I'll keep this open so we can compare both once i've commit that change